Makefile

```diff
@@ -5,7 +5,7 @@ build:
 	go build './...'
 
 test:
-	go get github.com/jpoles1/gopherbadger
+	go install github.com/jpoles1/gopherbadger@v2.4.0+incompatible
```

Comment:
This kind of sticks out to me 🤔

What's the +incompatible flag here?

Reply:
This is part of the packaging items, not actually needed here so I will remove it. The `+incompatible` means that the library is not setup as a go module.

Reply:
I grabbed the version information from the go.mod file that stores it with the additional tag on the version.